src/client/components/CommentFooter/Buttons.js

```diff
@@ -211,37 +212,38 @@ class Buttons extends React.Component {
             )}
           </a>
         </Tooltip>
-        <span
-          className={classNames('CommentFooter__count', {
-            'CommentFooter__count--clickable': upVotes.length > 0 || downVotes.length > 0,
-          })}
-          role="presentation"
-          onClick={this.handleShowReactions}
-        >
-          <Tooltip
-            title={
-              <div>
-                {downVotesPreview}
-                {downVotesMore}
-                {downVotes.length === 0 && (
-                  <FormattedMessage id="no_dislikes" defaultMessage="No dislikes" />
-                )}
-              </div>
-            }
+        {downVotes.length > 0 && (
+          <span
+            className={classNames('CommentFooter__count', {
+              'CommentFooter__count--clickable': upVotes.length > 0,
+            })}
+            role="presentation"
+            onClick={this.handleShowReactions}
           >
-            <FormattedNumber value={downVotes.length} />
-            <span />
-          </Tooltip>
-        </span>
-        <span className="CommentFooter__bullet" />
-        <span className="CommentFooter__payout">
-          <Tooltip title={<PayoutDetail post={comment} />}>
-            <USDDisplay
-              value={payout.cashoutInTime ? payout.potentialPayout : payout.pastPayouts}
-            />
-            <span />
-          </Tooltip>
-        </span>
+            <Tooltip
+              title={
+                <div>
+                  {downVotesPreview}
+                  {downVotesMore}
+                </div>
+              }
+            >
+              <FormattedNumber value={downVotes.length} />
+              <span />
+            </Tooltip>
+          </span>
+        )}
+        {payoutValue >= 0.01 && (
+          <>
```

Comment:
This `<>` is special syntax and requires babel plugin to transform it into `<React.Fragment>`. We should just use `<React.Fragment>` instead.